state-chain/pallets/cf-elections/src/lib.rs

```diff
@@ -316,21 +316,24 @@ pub mod pallet {
 		ElectoralUnsynchronisedState,
 		ElectoralUnsynchronisedSettings,
 		ElectoralSettings,
+		BlockNumber,
 	> {
 		pub unsynchronised_state: ElectoralUnsynchronisedState,
 		pub unsynchronised_settings: ElectoralUnsynchronisedSettings,
 		pub settings: ElectoralSettings,
+		pub block_shared_data_lifetime: BlockNumber,
 	}
 
 	#[cfg(feature = "runtime-benchmarks")]
-	impl<A: BenchmarkValue, B: BenchmarkValue, C: BenchmarkValue> BenchmarkValue
-		for InitialState<A, B, C>
+	impl<A: BenchmarkValue, B: BenchmarkValue, C: BenchmarkValue, D: BenchmarkValue> BenchmarkValue
+		for InitialState<A, B, C, D>
 	{
 		fn benchmark_value() -> Self {
-			InitialState::<A, B, C> {
+			InitialState::<A, B, C, D> {
 				unsynchronised_state: A::benchmark_value(),
 				unsynchronised_settings: B::benchmark_value(),
 				settings: C::benchmark_value(),
+				block_shared_data_lifetime: D::benchmark_value(),
```

Comment:
nit: - clearer to just name it the exact same (probably should've mentioned that before... I just corrected the liveness part)
```suggestion
				shared_data_reference_lifetime: D::benchmark_value(),
```